pom.xml

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.scalecube</groupId>
+    <artifactId>scalecube-benchmarks</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>ScaleCube/scalecube-benchmarks</name>
+
+    <description>benchmarks</description>
+    <inceptionYear>2015</inceptionYear>
+    <url>http://scalecube.io</url>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <prerequisites>
+        <maven>3.0.3</maven>
+    </prerequisites>
+
+    <scm>
+        <url>https://github.com/scalecube/scalecube</url>
+        <connection>scm:git:git@github.com:scalecube/scalecube-benchmarks.git</connection>
+        <developerConnection>scm:git:git@github.com:scalecube/scalecube-benchmarks.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>scalecube.io</id>
+            <name>The ScaleCube Project Contributors</name>
+            <email>info@scalecube.io</email>
+            <organization>The ScaleCube Project</organization>
+            <organizationUrl>http://scalecube.io/</organizationUrl>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <modules>
+        <module>benchmarks</module>
+        <module>example</module>
```

Comment:
Plz rename module `examples` to `benchmarks-examples`; plz rename moudle `benchmarks` to `benchmarks-api`.